Task 3/Decompression Algorithm

```diff
@@ -0,0 +1,139 @@
+package dev.benedikt.compression.sixpack
```

Comment:
Make a separate folder and rename the file with .kt extension

Reply:
Inline with your inference Folder structure maintained new pull request created.Hence closing the Pull Request